src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,11 +5,31 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private double armLength;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator`, `Truck` and `Bulldozer` classes. It's better to make `armLength` private and provide getters and setters for access.